test/mocks/rawAnalysisData.js

```diff
@@ -48,6 +49,7 @@ const mockRawAnalysisData = [
     pmFileName: '',
     analysisFileName: 'analysis-SF1624HT2018_9.pdf',
     roundIdList: '9',
```

Comment:
`ladokRoundIds ` and `roundIdList` has to be removed everywhere, in order to be sure tests work

Reply:
ladokRoundIds and roundIdList was removed from everywhere